style.css

```diff
@@ -382,6 +398,7 @@ ul.container3 {
 .lang {
   font-family: inherit;
   font-size: 20px;
+  color: white;
 }
 
 ul.points {
```

Comment:
- [ ] kindly make sure the list of languages and skills e.t.c are aligned properly as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/206457799-45d5eaef-e42e-4d80-b6e9-730f28bffbe1.png)  |  ![image](https://user-images.githubusercontent.com/74814780/206457863-c58b18ec-9fd1-45ea-81ce-ef4286d87296.png)